velox/exec/AssignUniqueId.cpp

```diff
@@ -33,7 +33,10 @@ AssignUniqueId::AssignUniqueId(
           planNode->id(),
           "AssignUniqueId"),
       rowIdPool_(std::move(rowIdPool)) {
-  VELOX_CHECK_LT(uniqueTaskId, kTaskUniqueIdLimit)
+  VELOX_USER_CHECK_LT(
```

Comment:
One question: why its user error now? Does the query have any issue with internal id assignment?

Reply:
This condition indicates that Velox received invalid query plan, hence, it is a user error. It is not related to the bug. I just fixed it because I noticed it.